packages/shared/src/components/search/SearchPanel/SearchPanel.tsx

```diff
@@ -203,7 +198,7 @@ export const SearchPanel = ({ className }: SearchPanelProps): ReactElement => {
               <div className="flex flex-1 flex-col">
                 <SearchPanelAction provider={SearchProviderEnum.Posts} />
                 <SearchPanelAction provider={SearchProviderEnum.Chat} />
-                {isGoogleSearchEnabled && (
+                {checkIsExtension() && (
```

Comment:
Should we consider something like this? (Not sure but would like to avoid re-render because func)
```
const isExtension = checkIsExtension();
```

Reply:
Add this?
```ts
const isExtension = !!process.env.TARGET_BROWSER;
```
As `checkIsExtension` is this
```ts
export const checkIsExtension = (): boolean => !!process.env.TARGET_BROWSER;
```

Reply:
Also works :) 
Not really sure why it's a func tbh

Just noticed we do the first one quite often already.

Reply:
Done. Going to test that it works in ext with this

Reply:
Works.